library/std/src/sys/unix/process/process_vxworks.rs

```diff
@@ -18,7 +18,7 @@ impl Command {
         needs_stdin: bool,
     ) -> io::Result<(Process, StdioPipes)> {
         use crate::sys::cvt_r;
-        const CLOEXEC_MSG_FOOTER: &'static [u8] = b"NOEX";
+        // const CLOEXEC_MSG_FOOTER: &'static [u8] = b"NOEX";
```

Comment:
This looks like a leftover from `process_unix.rs`, which is used to communicate an exec error from the fork to the parent process, and irrelevant to this vxwork implementation.
```suggestion
```